src/garage/sampler/on_policy_vectorized_sampler.py

```diff
@@ -106,13 +106,14 @@ def obtain_samples(self, itr, batch_size=None, whole_paths=True):
 
         while n_samples < batch_size:
             t = time.time()
-            policy.reset(dones)
+            policy.reset(completes)
```

Comment:
So just to be clear, a policy should always reset after a max path length is reached or if there is a genuine done signal @krzentner